service/javadsl/server/src/main/scala/com/lightbend/lagom/internal/javadsl/server/JavadslServerBuilder.scala

```diff
@@ -67,25 +69,31 @@ class JavadslServerBuilder @Inject() (environment: Environment, httpConfiguratio
   }
 
   /**
-   * Create a service info for the given interface.
-   *
-   * @param interface The interface to create the service info for.
-   * @return The service info.
+   * Create a service info for the given interfaces.
    */
-  def createServiceInfo(interface: Class[_]): ServiceInfo = {
-    if (classOf[Service].isAssignableFrom(interface)) {
-      val descriptor = ServiceReader.readServiceDescriptor(
-        environment.classLoader,
-        interface.asSubclass(classOf[Service])
-      )
-      new ServiceInfo(descriptor.name())
+  def createServiceInfo(primaryServiceInterface: Class[_], secondaryServices: Seq[Class[_]]): ServiceInfo = {
+    val interfaces = primaryServiceInterface +: secondaryServices
+    if (interfaces.forall(classOf[Service].isAssignableFrom)) {
+      val descriptors = interfaces.map { serviceInterface =>
+        ServiceReader.readServiceDescriptor(
+          environment.classLoader,
+          serviceInterface.asSubclass(classOf[Service])
+        )
+      }
+      val locatableServices = descriptors
+        .filter(_.locatableService())
+        .map { descriptor =>
+          descriptor.name() -> TreePVector.from(descriptor.acls().stream().collect(Collectors.toList()))
```

Comment:
Because on y first implementation of `ServiceInfo` acls was a `List` not a `PSequence`. Fixed.

Thanks!